app/gateway/upgrade/lts-upgrade.md

```diff
@@ -8,15 +8,200 @@ products:
+  - text: "Dual-cluster upgrade"
+    url: /gateway/upgrade/dual-cluster/
+  - text: "In-place upgrade"
+    url: /gateway/upgrade/in-place/
+  - text: "Rolling upgrade"
+    url: /gateway/upgrade/rolling/
+  - text: "{{site.base_gateway}} upgrade"
+    url: /gateway/upgrade/lts-upgrade/
 
 ---
 
+{{site.base_gateway}} supports direct upgrades between long-term support (LTS) versions of {{site.ee_product_name}}.
+This guide walks you through upgrading from {{site.ee_product_name}} 2.8 LTS to {{site.ee_product_name}} 3.4 LTS.
 
-@TODO: LTS Upgrade content
-pull content from https://docs.konghq.com/gateway/latest/lts-upgrade/
\ No newline at end of file
+There are three upgrade strategies available for the LTS to LTS upgrade: in-place, dual-cluster, and rolling.
+This guide nominates the best applicable strategy for each deployment mode that {{site.base_gateway}} supports. 
+Additionally, it lists some fundamental factors that play important roles in the upgrade process, and explains how to back up and recover data.
+
+This guide uses the following terms in the context of {{site.base_gateway}}:
+* **Upgrade**: The overall process of switching from an older to a newer version of {{site.base_gateway}}. 
+* **Migration**: The migration of your data store data into a new environment. 
+For example, the process of moving 2.8.x data from an old PostgreSQL instance to a new one for 3.4.x is referred to as database migration.
+
+To make sure your upgrade is successful, carefully review all the steps in this guide. It’s very important to understand all the preparation steps and choose the recommended upgrade path based on your deployment type.
+
+{:.warning}
+> **Caution**: The migration pattern described in this document can only happen between two LTS versions, {{site.ee_product_name}} 2.8 LTS and {{site.ee_product_name}} 3.4 LTS. If you apply this document to other release intervals, database modifications may be run in the wrong sequence and leave the database schema in a broken state. To migrate between other versions, see the [general upgrade guide](/gateway/upgrade/).
+
+## Prerequisites
+
+Read this document thoroughly to successfully complete the upgrade process, as it includes all the necessary operational knowledge for the upgrade.
+
+* Starting from 3.4, Cassandra is not supported. 
+If you're using Cassandra as your data store, migrate off of Cassandra first and upgrade second.
+Work with the Kong support team to migrate from Cassandra to PostgreSQL.
+* Review version compatibility between your platform version and the version of {{site.kong_gateway}} that you are upgrading to:
+  * [OS version](/gateway/version-support-policy/#supported-versions)
+  * [Kubernetes version and Helm prerequisites](/kic/version-support-policy/)
+  * [Hardware resources](/gateway/performance/resource-sizing-guidelines/)
+  * [Database and dependency versions](/gateway/third-party-support/)
+* If you're using decK, [upgrade it to the latest version](/deck/).
+
```

Comment:
```suggestion
{:.warning}
> Starting from {{site.base_gateway}} 3.4, Cassandra is not supported. 
>If you're using Cassandra as your data store, migrate off of Cassandra first and upgrade second.
>Work with the Kong support team to migrate from Cassandra to PostgreSQL.
```

Reply:
I felt like the original way you had it, it didn't feel like prerequisite but rather something that should be called out individually. Migrating to a different database felt different than upgrading deck.